lib/order.rb

```diff
@@ -0,0 +1,34 @@
+require_relative 'text'
+require_relative 'menu'
+
+class Order
+  include Menu
+
+  attr_accessor :contents
+
+  def initialize
+    @contents = []
+  end
+
+  def add_item(dish, quantity)
+    if display_menu.key?(dish)
+      @contents << dish
+      @contents << quantity
+    else
+      fail 'Item not on menu'
+    end
+  end
+
+  def order_total(total_price)
+    prices = []
+    order = Hash[@contents.each_slice(2).to_a]
+    order.each do |dish, quantity|
+    prices << display_menu[dish]*quantity
+    end
+    if total_price == prices.inject(:+)
```

Comment:
I'm a little confused as to why you are checking this. Plus, it seems a little odd that `order_total` returns `true` or `false`, when it sounds like it should return a number.